TedTalks/TedTalks/Controllers/Cells/TedTalkTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TedTalksTableViewCell.swift
+//  TedTalks
+//
+//  Created by Miguel Angel Askar Rodriguez on 4/08/21.
+//
+
+import UIKit
+
+class TedTalkTableViewCell: UITableViewCell {
+
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    
+    @IBOutlet weak var descriptionLabel: UILabel!
+    
+    @IBOutlet weak var thumbnailmageView: UIImageView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
+    
+    func fill(with tedTalk: TedTalk) {
+        titleLabel.text = tedTalk.title
+        descriptionLabel.text = tedTalk.description
+        guard let thumbnailURL = tedTalk.thumbnail else { return }
```

Comment:
Let's clear the image from the ImageView if this fail since if a given talk doesn't have a thumbnail when the cell is reused, the previous loaded image will be kept.

Reply:
Ok, fixed